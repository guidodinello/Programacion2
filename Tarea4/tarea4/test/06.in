# Implementar imprimirPorNiveles
recorridaPorNiveles
insertarEnBinario (5,1.0)
recorridaPorNiveles
insertarEnBinario (8,1.0)
recorridaPorNiveles
insertarEnBinario (2,1.0)
recorridaPorNiveles
insertarEnBinario (6,1.0)
recorridaPorNiveles
insertarEnBinario (1,1.0)
recorridaPorNiveles
insertarEnBinario (7,1.0)
recorridaPorNiveles
insertarEnBinario (70,1.0)
insertarEnBinario (10,1.0)
insertarEnBinario (20,1.0)
insertarEnBinario (90,1.0)
insertarEnBinario (80,1.0)
insertarEnBinario (15,1.0)
insertarEnBinario (25,1.0)
insertarEnBinario (75,1.0)
insertarEnBinario (85,1.0)
recorridaPorNiveles
Fin
