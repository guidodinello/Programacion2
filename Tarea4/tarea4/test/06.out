1>#  Implementar imprimirPorNiveles.
2>
3>Insertado.
4>5 
5>Insertado.
6>8 | 5 
7>Insertado.
8>2 8 | 5 
9>Insertado.
10>6 | 2 8 | 5 
11>Insertado.
12>1 6 | 2 8 | 5 
13>Insertado.
14>7 | 1 6 | 2 8 | 5 
15>Insertado.
16>Insertado.
17>Insertado.
18>Insertado.
19>Insertado.
20>Insertado.
21>Insertado.
22>Insertado.
23>Insertado.
24>15 25 75 85 | 20 80 | 7 10 90 | 1 6 70 | 2 8 | 5 
25>Fin.
